src/locales/tr/translation.json

```diff
@@ -0,0 +1,162 @@
+      "installation_instructions_description": "Bu oyunu kurmak için ek adımlar gerekiyor",
+      "online_fix_instruction": "OnlineFix oyunlarını ayıklamak için parola gerekiyor. Gerekli olduğunda bu parolayı kullanın:",
+      "dodi_installation_instruction": "Dodi installerını açtığınızda, kurulumu başlatmak için bu tuşa basın <0 />:",
+      "dont_show_it_again": "Tekrar gösterme",
+      "copy_to_clipboard": "Kopyala",
+      "copied_to_clipboard": "Kopyalandı",
+      "got_it": "Tamam"
+    },
+    "activation": {
+      "title": "Hydra'yı aktif et",
+      "installation_id": "Kurulum ID'si:",
+      "enter_activation_code": "Aktifleştirme kodunuzu girin",
+      "message": "Bunu nerede soracağınızı bilmiyorsanız, buna sahip olmamanız gerekiyor.",
+      "activate": "Aktif et",
+      "loading": "Yükleniyor…"
+    },
+    "downloads": {
+      "resume": "Devam et",
+      "pause": "Duraklat",
+      "eta": "Bitiş {{eta}}",
+      "paused": "Duraklatıldı",
+      "verifying": "Doğrulanıyor…",
+      "completed_at": "{{date}} tarihinde tamamlanacak",
+      "completed": "Tamamlandı",
+      "cancelled": "İptal edildi",
+      "download_again": "Tekrar indir",
+      "cancel": "İptal et",
+      "filter": "Yüklü oyunları filtrele",
+      "remove": "Kaldır",
+      "downloading_metadata": "Metadata indiriliyor…",
+      "checking_files": "Dosyalar kontrol ediliyor…",
+      "starting_download": "İndirme başlatılıyor…",
+      "deleting": "Installer siliniyor…",
+      "delete": "Installer'ı sil",
+      "remove_from_list": "Kaldır",
+      "delete_modal_title": "Emin misiniz?",
+      "delete_modal_description": "Bu bilgisayarınızdan tüm kurulum dosyalarını silecek",
+      "install": "Kur"
+    },
+    "settings": {
+      "downloads_path": "İndirme yolu",
+      "change": "Güncelle",
+      "notifications": "Bildirimler",
+      "enable_download_notifications": "Bir indirme bittiğinde",
+      "enable_repack_list_notifications": "Yeni bir repack eklendiğinde",
+      "telemetry": "Telemetri",
+      "telemetry_description": "Anonim kullanım istatistiklerini aktifleştir"
+    },
+    "notifications": {
+      "download_complete": "İndirme tamamlandı",
+      "game_ready_to_install": "{{title}} kuruluma hazır",
+      "repack_list_updated": "Repack listesi güncellendi",
+      "repack_count_one": "{{count}} yeni repack eklendi",
+      "repack_count_other": "{{count}} yeni repack eklendi"
+    },
+    "system_tray": {
+      "open": "Hydra'yı aç",
+      "quit": "Çık"
+    },
+    "game_card": {
+      "no_downloads": "İndirme mevcut değil"
+    },
+    "binary_not_found_modal": {
+      "title": "Programlar yüklü değil",
+      "description": "Sisteminizde Wine veya Lutris çalıştırılabiliri bulunamadı",
+      "instructions": "Oyunları düzgün şekilde çalıştırmak için Linux distronuza bunlardan birini nasıl yükleyebileceğinize bakın"
+    },
+    "modal": {
+      "close": "Kapat tuşu"
+    }
+  }
+  
```

Comment:
Newline at the ends missing

Reply:
Newline's still missing

Reply:
There's a empty line at the end of the file, what's wrong?